modules/ensemble/lib/widget/input/form_time.dart

```diff
@@ -112,7 +113,7 @@ class TimeController extends FormFieldController with HasTextPlaceholder {
   TimeOfDay? value;
   TimeOfDay? initialValue;
   EnsembleAction? onChange;
-  bool useIOSStyleTimePicker = Platform.isIOS;
+  bool useIOSStyleTimePicker = kIsWeb ? false : Platform.isIOS;
```

Comment:
if someone is running the webapp on safari on iOS, it seems ok to show them the iOSStyleTimePicker. Why make it false?

Reply:
As i showed yesterday, web build was failing, it was due to that `Platform` library was not available on Web, so I put a check on if its not web then check.
But yeah I can do the following scenario:

- if it's web, check the if on Apply platform then true otherwise false.
- if not web then check native platform, if iOS then true else false..